Compressonator/Applications/CompressonatorCLI/CMakeLists.txt

```diff
@@ -46,6 +46,21 @@ target_include_directories(CompressonatorCLI-bin
                            ../../CMP_Core/source
                            /usr/include/OpenEXR)
 
+find_path(ILMBASE_INCLUDE_DIR OpenEXR/half.h)
```

Comment:
There is no need to do this by yourself. Either use FindModule for pkg-config, which will give you ${OpenEXR_LIBRARIES} and ${OpenEXR_INCLUDE_DIRs}. I resolved this in my merge request: https://github.com/GPUOpen-Tools/Compressonator/pull/86
I could you please see if this patch works for you?